tasks/Makefile.hugo

```diff
@@ -10,6 +10,6 @@ hugo/run: components/build
 hugo/build: components/build
 	@[ "$(HUGO_PUBLISH_DIR)" != "/" ] || (echo "Invalid HUGO_PUBLISH_DIR=$(HUGO_PUBLISH_DIR)"; exit 1)
 	rm -rf $(HUGO_PUBLISH_DIR)
-	$(DOCKER_RUN) hugo --templateMetrics --stepAnalysis --config $(HUGO_CONFIG)
+	$(DOCKER_RUN) hugo --templateMetrics --trace trace.out --config $(HUGO_CONFIG)
```

Comment:
```suggestion
	$(DOCKER_RUN) hugo --templateMetrics --trace /dev/stdout --config $(HUGO_CONFIG)
```

Reply:
good point! I'll remember to just send outputs to some stream, rather than a file, if I don't have an explicit plan for how to exfiltrate the file from the Docker container.